Tests/Tests.swift

```diff
@@ -16,8 +16,7 @@ class Tests: XCTestCase {
     func testEmptyView() {
         let url = self.url(forResource:"View", withExtension: "xib")
         do {
-            let file = try XibFile(url: url)
-            print(file.document.targetRuntime)
```

Comment:
Hi, why removing all print code?
This allow to debug easily with debug point and it is not part of font feature

Reply:
I think the tests logs are more readable if we strip off all the debug statements. It's always possible to add them locally when testing and for debugging purposes.
If you don't agree with me, I can just leave them in place.

Reply:
that's only simple print, not really logger with level ;(
so some are info level and userfull like   print(name) to print storyboard tested
but yes some are too verbose and must not be always displayed, some need to be rewritten

But now I could use it in CI to analyse without opening an Xcode
So I am to keep it until have nothing better

Reply:
Ok no worries. I'll update my pull request and drop this commit.